packages/extension/src/newtab/ShortcutLinksModal.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FormEventHandler, MutableRefObject, ReactElement } from 'react';
+import classNames from 'classnames';
+import { ModalHeader } from '@dailydotdev/shared/src/components/modals/common';
+import { ModalProps } from '@dailydotdev/shared/src/components/modals/StyledModal';
+import { ResponsiveModal } from '@dailydotdev/shared/src/components/modals/ResponsiveModal';
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import XIcon from '@dailydotdev/shared/icons/x.svg';
+import UserIcon from '@dailydotdev/shared/icons/user.svg';
+import SitesIcon from '@dailydotdev/shared/icons/sites.svg';
+import { CardSelection } from './CardSelection';
+import { LinksForm } from './LinksForm';
+
+const getIconClassName = (isActive?: boolean) =>
+  classNames('w-8 h-8', !isActive && 'text-theme-label-tertiary');
+
+interface CustomLinksModalProps extends ModalProps {
+  errors?: Record<string | number, string>;
+  onSubmit: FormEventHandler<HTMLFormElement>;
+  onShowPermission: () => unknown;
+  onShowCustomLinks: () => unknown;
+  onShowTopSitesClick: () => unknown;
+  isManual?: boolean;
+  hasTopSites?: boolean;
+  links?: string[];
+  formRef: MutableRefObject<HTMLFormElement>;
+  onRevokePermission?: () => Promise<unknown>;
+  onIsManual?: (isManual: boolean) => unknown;
+}
+
+export default function CustomLinksModal({
+  isManual,
+  hasTopSites,
+  links,
+  style,
+  formRef,
+  errors,
+  onRevokePermission,
+  onShowPermission,
+  onShowTopSitesClick,
+  onShowCustomLinks,
+  onRequestClose,
+  onSubmit,
+  ...props
+}: CustomLinksModalProps): ReactElement {
+  return (
+    <ResponsiveModal
+      {...props}
+      padding={false}
+      contentClassName="max-h-[40rem]"
+      style={{ ...style, content: { maxHeight: '40rem' } }}
+    >
+      <form ref={formRef} onSubmit={onSubmit}>
+        <ModalHeader>
+          <h3 className="font-bold typo-title3">Shortcuts</h3>
+          <div className="flex-1" />
+          <Button className="mr-3 btn-primary" buttonSize="small" type="submit">
+            save changes
```

Comment:
This is my first very useful comment 😅  I noticed a lower case letter in "Save changes".

Reply:
Great spot! But that was actually part of the design spec 😅

So happy to see you take part in the review!

Reply:
Then I am sorry! I'll double check the next time 😄

Reply:
Actually, I just checked and I was wrong! It should have been on caps! You were right! My bad! Thank you for the additional pair of eyes.